app/Http/Controllers/AboutController.php

```diff
@@ -4,18 +4,35 @@
 
 use App\Models\Contributor;
 use Illuminate\Contracts\View\View;
+use Illuminate\Support\Facades\Cache;
+use JsonException;
 
 final readonly class AboutController
 {
+    private const DAY = 60 * 60 * 24;
+
     public function __invoke(): View
     {
-        $contributors = array_map(
-            fn (array $item) => new Contributor(...$item),
-            json_decode(file_get_contents(__DIR__.'/../../../contributors.json') ?: '{}', true)['contributors'] ?? [],
-        );
+        $contributors = Cache::remember('contributors', self::DAY, fn () => $this->getContributors());
```

Comment:
```suggestion
        $contributors = Cache::remember('contributors', self::DAY, $this->getContributors(...));
```

Reply:
Since PHP 8.1 https://stitcher.io/blog/new-in-php-81#first-class-callable-syntax-rfc